config/routes.rb

```diff
@@ -509,10 +509,11 @@
   #### I18N ####
 
   # should stay below the main works mapping
-  resources :languages do
+  resources :languages, except: [:show] do
     resources :works
     resources :admin_posts
   end
+  get "/languages/:id", to: redirect("/languages/%{id}/works", status: 302)
```

Comment:
I think it would be more obvious that this route belongs to `languages` if we use the `member` block like
```suggestion
  resources :languages, except: [:show] do
    resources :works
    resources :admin_posts

    member do
      get :show, to: redirect("/languages/%{id}/works", status: 302)
    end
  end
```

Reply:
Doesn't appear to work. Or more exactly, it works too well, changing all get routes on /languages to a redirect.

I would say the `get "/languages/:id", to:`is good enough as is? It's consistent with [official documentation(https://guides.rubyonrails.org/routing.html#redirection).